code/modules/mob/living/carbon/human/npc/dungeon_npc.dm

```diff
@@ -0,0 +1,379 @@
+		armor = /obj/item/clothing/cloak/tabard/crusader/tief
+		neck = /obj/item/clothing/neck/roguetown/coif
+		gloves = /obj/item/clothing/gloves/roguetown/chain/iron
+		pants = /obj/item/clothing/under/roguetown/chainlegs
+		wrists = /obj/item/clothing/wrists/roguetown/bracers/leather/advanced
+		shoes = /obj/item/clothing/shoes/roguetown/armor
+		r_hand = /obj/item/rogueweapon/spear
+
+		H.STASTR = 14
+		H.STASPD = 10
+		H.STACON = 13
+		H.STAEND = 14
+		H.STAINT = 8
+		H.STAPER = 15
+
+/mob/living/carbon/human/species/human/northern/dungeon_base/warrior/use_combat_abilities()
+	if(src.stat != CONSCIOUS || world.time < next_cast)
+		return
+
+	next_cast = world.time + 100
+
+	var/list/enemies_in_range = list()
+	for(var/mob/living/L in view(3, src))
+		if(L != src && L.stat != DEAD && (L.faction == null || disjoint_lists(L.faction, src.faction)))
+			enemies_in_range += L
+
+	if(!enemies_in_range.len)
+		return
+
+	var/mob/living/target = pick(enemies_in_range)
+
+	spawn(50) // 10 SECONDS
+		if(target && get_dist(src, target) <= 3)
+			src.say("Watch your steps!")
+			warrior_strike(src, target)
+
+
+// 2. Paladin
+/mob/living/carbon/human/species/human/northern/dungeon_base/paladin/after_creation()
+	..()
+	equipOutfit(new /datum/outfit/job/roguetown/npc/dungeon_paladin)
+
+/datum/outfit/job/roguetown/npc/dungeon_paladin
+	name = "Dungeon Paladin"
+
+	pre_equip(mob/living/carbon/human/H)
+		..()
+		head = /obj/item/clothing/head/roguetown/helmet/skullcap
+		mask = /obj/item/clothing/mask/rogue/facemask
+		armor = /obj/item/clothing/cloak/tabard/crusader/tief
+		shirt = /obj/item/clothing/suit/roguetown/armor/gambeson/heavy
+		armor = /obj/item/clothing/suit/roguetown/armor/chainmail/hauberk
+		neck = /obj/item/clothing/neck/roguetown/gorget
+		pants = /obj/item/clothing/under/roguetown/chainlegs
+		shirt = /obj/item/clothing/suit/roguetown/armor/chainmail/iron
+		armor = /obj/item/clothing/cloak/tabard/crusader/tief
+		neck = /obj/item/clothing/neck/roguetown/coif
+		gloves = /obj/item/clothing/gloves/roguetown/chain/iron
+		pants = /obj/item/clothing/under/roguetown/chainlegs
+		wrists = /obj/item/clothing/wrists/roguetown/bracers/leather
+		shoes = /obj/item/clothing/shoes/roguetown/armor
+		r_hand = /obj/item/rogueweapon/sword
+		l_hand = /obj/item/rogueweapon/shield/wood
+
+		H.STASTR = 15
+		H.STASPD = 15
+		H.STACON = 15
+		H.STAEND = 14
+		H.STAINT = 8
+		H.STAPER = 15
+
+
+/mob/living/carbon/human/species/human/northern/dungeon_base/paladin/use_combat_abilities()
+	if(src.stat != CONSCIOUS || world.time < next_cast)
+		return
+
+	next_cast = world.time + 300
+
+	var/list/enemies_in_range = list()
+	for(var/mob/living/L in view(3, src))
+		if(L != src && L.stat != DEAD && (L.faction == null || disjoint_lists(L.faction, src.faction)))
+			enemies_in_range += L
+
+	if(!enemies_in_range.len)
+		return
+
+	var/mob/living/target = pick(enemies_in_range)
+
+	spawn(50) // 5 SECONDS
```

Comment:
so I use spawn a fair bit, but this ain't it. Unless you are fine with them just rolling these whenever, it should likely use either some form of pickweight with internal cds for the abilities, or gating the abilities behind some form of requirement. I don't really get what the intention of the spawn here even is? Are they like suppose to be casting the spell on the move? If so a timer would suffice.